src/viam/utils.py

```diff
@@ -1,12 +1,25 @@
 import asyncio
 import contextvars
 import functools
-import numpy as np
+
+try:
+    import numpy as np
+    from numpy.typing import NDArray
+except ImportError:
+    import warnings
+
+    warnings.warn(
```

Comment:
this warning will print on import of the utils package, which will likely not involve the MLModel at all - I would consider moving the specific numpy utils out to a different file

Reply:
yep true 😱